app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+
+    if len(split_command) != 3 or split_command[0] != "cp":
+        print("Invalid command")
+        return
+
+    file_to_copy = split_command[1]
+    copied_file = split_command[2]
+
+    if file_to_copy == copied_file:
+        return
```

Comment:
It might be helpful to inform the user that the source and destination files are the same, which is why no action is taken. This can prevent confusion.